terraform/environments/production/portal.tf

```diff
@@ -420,7 +420,8 @@ module "web" {
   image      = "web"
   image_tag  = var.image_tag
 
-  scaling_horizontal_replicas = 2
+  scaling_horizontal_replicas     = 2
+  scaling_max_horizontal_replicas = 4
```

Comment:
This should not kick in right now but I'm adding autoscaling so that we can get operational experience with it before we actually need it.